pkg/kwok/controllers/node_controller.go

```diff
@@ -413,8 +413,9 @@ func (c *NodeController) LockNode(ctx context.Context, node *corev1.Node) error
 		return nil
 	}
 
-	logger.Info("Delayed play stage",
+	logger.Debug("Delayed play stage",
 		"delay", delay,
+		"delayString", delay.String(),
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/122f36ce87128371d2b9bbcff43837d764b35e37/pkg/log/logger.go#L58-L68

``` golang
			if a.Value.Kind() == slog.DurationKind {
				if t, ok := a.Value.Any().(time.Duration); ok {
					return slog.Any(a.Key, DurationFormat{
						Nanosecond: int64(t),
						Human:      t.String(),
					})
				}
			}
```

I'm thinking deeply about this issue, and adding in this is more appropriate.